### DISCUSSION TITLE
aspcore Identiny Block user

### DISCUSSION THREAD

**Discussion by EvgenyPrikhodko** [dotnet/aspnetcore#25890] [General] [answered]
Identity doesn't have a standard user blocking?

**Comment by SteveSandersonMS** [comment, marked as answer]
There are some suggestions about how to block/ban users here:

 * https://stackoverflow.com/questions/22652118/disable-user-in-aspnet-identity-2-0
 * http://bekenty.com/disable-user-asp-net-core-identity-2-0/

Do you think either of these might work for your scenario?

**Reply by EvgenyPrikhodko** [reply]
Thanks, let's try.
It seems to me that this functionality should be included in Identity by default

**Comment by davidfowl** [comment]
cc @HaoK @blowdart

**Comment by blowdart** [comment]
Correct, it doesn't. You need to expand it following the approach in Steve's links.
